packages/flutter_localizations/lib/src/l10n/generated_material_localizations.dart

```diff
@@ -6729,9 +6729,24 @@ class MaterialLocalizationEnAu extends MaterialLocalizationEn {
   @override
   String get keyboardKeyNumpadEnter => 'Num enter';
 
+  @override
+  String get timePickerDialHelpText => 'SELECT TIME';
```

Comment:
This doesn't look correct: it should be sentence case, right?

Reply:
From what I understand, the locales other than `MaterialLocalizationEn` will be up to date once we request translations?

Reply:
Probably these will just get deleted again once the translations are done, since I'm sure they're identical to en_US.  I just noticed that it was adding new values here that weren't matching the updated values in the en_US locale.

Reply:
And I was concerned that because of that, the translators for those locales would just leave them as-is.